Sources/Carpaccio/ImageMetadata.swift

```diff
@@ -591,6 +669,149 @@ public enum ImageOrientation: String, Codable {
     }
 }
 
+public enum FlashMode: String, Codable {
```

Comment:
My only nitpick is really the inconsistent capitalisation for these cases. Any particular significance to the combination of capitalisation applied?

Reply:
Yeah, I went back and forth on that. It was mostly for displaying what the values meant. There's probably a better way to do that, so I'll go with the same kebab-case used everywhere else

Reply:
Switched to kebab-case, and added a `humanReadableFlashMode`. It's english only, so if we want to get rid of it, that's fine too!

Reply:
I actually meant to whinge about the odd combo of sentence cased phrases with a comma separating them (commented on this now over at https://github.com/mz2/Carpaccio/pull/13#discussion_r707659667), but this separation of the encoding + human readable string also makes sense 😄